### ISSUE TITLE
Accessibility_Testing_Link_Core

### ISSUE THREAD

**Issue by Wemitha1234** [SAP/fundamental-ngx#6208]
Accessibility: Link-Core Component (There are couple of accessibility issues found while testing, so i have attached error document for you reference.) Screenshot:

[Link_Core_ARIBA Controls.docx](https://github.com/SAP/fundamental-ngx/files/6920694/Link_Core_ARIBA.Controls.docx)

Issue 1

Priority:	Medium
Checkpoint:	ACC-264

Observed Behavior:
Emphasized information about links is not read. 
Only the link text emphasized is read.  

The speech output is:
Standard Link Link  
Emphasized Link Link 

Expected Behavior:
Ideally, emphasized information about the UI should be read out.  

Screenshot:
![image](https://user-images.githubusercontent.com/88035260/127949583-c48381e7-2922-433d-b609-2b40b995b062.png)

Issue 2

Priority:	Medium
Checkpoint:	ACC-261

Observed Behavior:
The color contrast ratio for focus indication does not satisfy the minimum contrast ratio of 4.5:1 in standard theme. 

Expected Behavior:
Ideally, in standard theme, the contrast ratio should be equal to or more than 4.5:1   

Screenshot:

Screenshot:
![image](https://user-images.githubusercontent.com/88035260/127949610-e6831cca-159b-40ef-9c8d-8fef6929329f.png)

Issue 3

Priority:	Medium
Checkpoint:	ACC-262

Observed Behavior:
In default theme, underline is visible when links are focused.  
Whereas in high contrast black/high contrast white themes, underlines are not visible when links are accessed.  

Expected Behavior:
Ideally, underline should appear when links are focused. 

Screenshot 1:
![image](https://user-images.githubusercontent.com/88035260/127949634-333f3d85-8393-4e08-946f-6b130a9123f0.png)

Screenshot 2:
![image](https://user-images.githubusercontent.com/88035260/127949647-b7b16efb-c195-4806-8ead-4ca68ca964f9.png)

Screenshot 3:
![image](https://user-images.githubusercontent.com/88035260/127949662-dffbac86-900d-4a63-b642-54ec4cf26dad.png)

Screenshot 4:
![image](https://user-images.githubusercontent.com/88035260/127949677-0dc0532e-4658-43ec-988a-b00c127147a3.png)

Screenshot 5:
![image](https://user-images.githubusercontent.com/88035260/127949689-1639cf46-5c3c-43e4-861e-29d7368074fb.png)

Issue 4

Priority:	Medium
Checkpoint:	ACC-261

Observed Behavior:
In high contrast white theme, focus indication provided for Inverted Link does not satisfy the minimum contrast ratio of 7:1 

Expected Behavior:
Ideally, in high contrast white theme, the contrast ratio should be equal to or more than 7:1

Screenshot:
![image](https://user-images.githubusercontent.com/88035260/127949713-1b8480ca-12c5-4941-8e27-c41dda53442a.png)

**Comment by Lokanathan-k** [comment]
Issue 1: fixed
Issue 2: not repordouceable
<img width="478" alt="Screenshot 2021-08-17 at 9 28 43 AM" src="https://user-images.githubusercontent.com/53534379/130553286-8646acc4-7116-44db-8722-205b7762ee08.png">
issue 3: Working as expected
<img width="1177" alt="Screenshot 2021-08-24 at 9 25 48 AM" src="https://user-images.githubusercontent.com/53534379/130553439-32fd713f-30c8-4df9-a419-867d9db9c477.png">
Issue 4: fixed
